libs/blocks/media/media.js

```diff
@@ -30,9 +30,10 @@ function decorateAvatar(el) {
 function decorateQr(el) {
   const text = el.querySelector('.text');
   if (!text) return;
-  const appStore = text.children[(text.children.length - 1)];
-  const googlePlay = text.children[(text.children.length - 2)];
   const qrImage = text.children[(text.children.length - 3)];
```

Comment:
Is it certain that all these elements exist? Otherwise, it wouldn't hurt to use optional chaining when trying to add classes below.

Reply:
Added a early return as @robert-bogos suggested